rpc/core/status.go

```diff
@@ -16,6 +16,23 @@ import (
 // hash, app hash, block height and time.
 // More: https://docs.tendermint.com/master/rpc/#/Info/status
 func Status(ctx *rpctypes.Context) (*ctypes.ResultStatus, error) {
+	var (
+		earliestBlockMeta     *types.BlockMeta
+		earliestBlockHash     tmbytes.HexBytes
+		earliestAppHash       tmbytes.HexBytes
+		earliestBlockTimeNano int64
+	)
+	earliestBlockHeight := blockStore.Base()
+	if earliestBlockHeight == 0 {
```

Comment:
can't we make this a default in blockStore, so we don't have to write such conditionals

Reply:
Yep, done